experimental/cmd/as7262/main.go

```diff
@@ -40,30 +35,24 @@ func mainImpl() error {
 	}
 	defer bus.Close()
 
-	// Create a new power sensor a sense with default options of 100 mΩ, 3.2A at
-	// address of 0x40 if no other address supplied with command line option.
+	// Create a spectrum sensor.
 	sensor, err := as7262.New(bus, &as7262.Opts{Gain: as7262.G16x})
 	if err != nil {
 		return fmt.Errorf("failed to open new sensor: %v", err)
 	}
 
-	// Read values from sensor every second.
+	// Create a ticker to read values from sensor every second.
 	everySecond := time.NewTicker(time.Second).C
-	var quit = make(chan os.Signal)
 	defer sensor.Halt()
-	signal.Notify(quit, syscall.SIGTERM)
-	signal.Notify(quit, syscall.SIGINT)
 
 	fmt.Println("ctrl+c to quit")
 
 	senseTime := time.Millisecond * 300
 
 	for {
 		select {
-		case <-quit:
-			return nil
 		case <-everySecond:
-			spectrum, err := sensor.Sense(100*physic.MilliAmpere, senseTime)
+			spectrum, err := sensor.Sense(12500*physic.MicroAmpere, senseTime)
```

Comment:
As a side note, I'd like to have all types in physic implement [flag.Value](https://golang.org/pkg/flag/#Value), which means implementing Set(), so CLI tools could accept values as strings more easily.

Reply:
I can definitely see value in that, it would also make for some more pipe-able tools.